assets/js/editor/Editor.tsx

```diff
@@ -189,8 +190,16 @@ export default function Editor({ source, adaptor, onChange, metadata }: EditorPr
       // ensure the editor has focus
       editor.focus();
     };
+    
+    // This is a temporary (and poor) solution to make the editor respond to resizing
+    listeners.current.updateLayout = (e: Event) => {
```

Comment:
I will need to sit and figure this out at some stage, but not necessarily in this PR (or even for the first release, it works better than I thought it would)